scripts/pytest-tests.sh

```diff
@@ -35,5 +35,27 @@ cd "$SRCDIR/test/python" && source ./venv/bin/activate
 run_tests << 'END'
 
 tests/cli_controller
+#tests/replica
```

Comment:
what!! the tests were not running on CI??? how did that happen??

Reply:
Seems they were removed by accident, please uncomment the "#" and let's run them all

Reply:
I think i got these changes while i was rebasing. I think it was your PR for allowed_hosts.  Not sure though. Shall i uncomment them all?

Reply:
Done